Add White Text and Highlighting Options

wmcconnell opened the pull request:
To accommodate the RTO hero design, added the ability to have white text over a hero image.

Changes proposed in this pull request:

- New ACF Pro field for choosing black or white text. This field only shows when a highlight style of 'none' has been chosen.
- Add a new 'white' highlight option for the hero title. This is described in the XD file, but was overlooked during the initial creation of the hero
- Updated the Hero template to accommodate these new field settings.

zainabD commented on templates-global/hero.php:
```diff
@@ -22,17 +23,43 @@
+} else {
+	$title_color_class = '';
+}
+
```
'black' option is missing here but was added to the ACF json file. 
`elseif ( 'black' == $title_color && 'none' == $hero_highlight || empty( $hero_highlight ) )`

Also, I would remove the other side of the condition `('none' == $hero_highlight || empty( $hero_highlight ))`, since the text color option won't show unless the $hero_highlight is none anyway.

zainabD commented on templates-global/hero.php:
```diff
@@ -22,17 +23,43 @@
 		<h1 class="col-md-8">
-			<span class="<?php echo $hero_highlight; ?>"><?php echo wp_kses( $hero_title, wp_kses_allowed_html( 'strip' ) ); ?></span>
+			<span class="<?php echo $hero_highlight; ?> <?php echo $title_color_class; ?>"><?php echo wp_kses( $hero_title, wp_kses_allowed_html( 'strip' ) ); ?></span>
 		</h1>
```
Since it's either **$hero_highlight** or **$title_color_class**.
We can create one var called **$hero_title_class**, by default it takes the value of **$hero_highlight**, and would take the value of the **$title_color_class** if it went through the if condition `if ( 'white' == $title_color) or black`.

wmcconnell commented:
I've updated the logic based on Zainab's feedback, and cleaned up the fields a bit to remove stuff I was doing to 'improve' the hero (I'll add that all back in a proper, dedicated pull request at a later date). I renamed some fields in the ACF group, but I think those are just labels and shouldn't affect the actual performance.
